tldw_Server_API/app/services/admin_settings_service.py

```
@@ -82,3 +82,42 @@ async def set_notes_title_settings(payload: NotesTitleSettingsUpdate) -> dict[st
+    "api_key_age": {"weight": 2, "cap": 25},
+    "failed_logins": {"weight": 1, "cap": 20},
+    "suspicious_activity": {"weight": 4, "cap": 20},
+}
+
+# In-memory store — persisted across requests within a single process
+_risk_weights: dict[str, Any] | None = None
+
+
+async def get_risk_weights() -> dict[str, Any]:
+    """Return the current risk weight configuration."""
+    global _risk_weights
```

Ruff (0.15.7)

[warning] Using the global statement to update `_risk_weights` is discouraged

(PLW0603)